thirdeye/pom.xml

```diff
@@ -22,17 +22,18 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>org.apache.pinot.thirdeye</groupId>
   <artifactId>thirdeye</artifactId>
-  <version>${revision}${sha1}</version>
```

Comment:
I think this is needed for the travis build!
+@vincentchenjl  FYI

Reply:
I'll restore this. This has nothing to do with the current refactor.

But it seems like the build worked fine. As such haven't really found versions to be used in TE. Also, the docker images uses its own versioning.